core/tests/puppeteer-acceptance-tests/utilities/user/curriculum-admin.ts

```diff
@@ -838,6 +839,23 @@ export class CurriculumAdmin extends BaseUser {
     }
   }
 
+  /**
+   * Function to close editor navigation dropdown. Can be done by clicking
+   * on the dropdown toggle.
+   */
+  async closeEditorNavigationDropdown(): Promise<void> {
+    try {
+      await this.page.waitForSelector(dropdownToggleIcon, {
```

Comment:
Should the function name be updated to end with "OnMobile", then, and the "if mobile" should come in the top-level test since it determines whether the user performs the action or not?

Reply:
Done